### ISSUE TITLE
Does it support automatically turning return value into oneshot channels and back?

### ISSUE THREAD

**Issue by vi** [Restioson/spaad#6]
In README and other examples I don't see handlers that return values.

Can spaad do something like [this](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=4c02d55ffab539ad8b4eeb12048ac62a)?

**Comment by Restioson** [comment]
Yes, it can. Simply write the return type and return at the end of the function and it should work.

**Comment by vi** [comment]
Maybe README should feature this then?

**Comment by vi** [comment]
Verified that my example can be adapted to Spaad with minimal changes.

When will there be a `Tokio 1`-enabled beta release on crates.io? Currently released `0.4.0-beta.2` does not seem to work, unlike the git version.

**Comment by Restioson** [comment]
Git version just needs an update to latest xtra release and then it should be all set. I'll get to that soon, unfortunately right now I'm just working through some university coursework backlog after being sick.

**Comment by Restioson** [comment]
Done - see spaad 0.4.0
